routes/lectureRoutes.js

```diff
@@ -72,14 +71,28 @@ router.patch("/:id", /*requireLogin,*/ async (req, res) => {
 });
 
 
+/**
+ * Get all lectures from a specific section id
+ * @param {string} sid - section id
+ * @returns {object} - lectures
+ */
+router.get("/getall/:sid", async (req, res) => {
```

Comment:
Wrong route name! Should just be `GET /lectures/:id`

Reply:
Yes we have fixed it